### ISSUE TITLE
chdir option does not interact correctly with file passed as argument to precious

### ISSUE THREAD

**Issue by oschwald** [houseabsolute/precious#25]
When using a linter with the `chdir` option enabled, `precious` does not strip off the directory before passing the filename to the linter. For instance (with slight modifications):

```
$ precious a/b.go

💍 Tidying paths passed on the Cli (recursively)
💥 error golangci-lint: a/b.go
Error when tidying files:
  ▶ a/b.go [commands.golangci-lint]
    Failed to execute command `golangci-lint run --fix a/b.go`: Got unexpected exit code 7 from `golangci-lint run --fix a/b.go`. Stderr was level=error msg="[linters context] typechecking error: stat /home/greg/go/a/a/b.go: directory not found"
```

Using a config such as:

```
[commands.golangci-lint]
type = "both"
cmd = [ "golangci-lint", "run" ]
chdir = true
tidy_flags = [ "--fix" ]
ok_exit_codes = 0
include = [ "**/*.go" ]
```

**Comment by autarch** [comment]
Thanks for the bug report! This is fixed in v0.1.5.

**Comment by oschwald** [comment]
Thank you!
